packages/compass-components/src/components/document-list/document.tsx

```diff
@@ -138,7 +129,7 @@ const HadronDocument: React.FunctionComponent<{
         data-testid="hadron-document"
         data-id={document.uuid}
       >
-        <AutoFocusContext.Provider value={autoFocus}>
+        <AutoFocusContext.Provider value={editing ? autoFocus : null}>
```

Comment:
Was this causing a bug previously? Seems like it could have been messing up the focus state if we auto focused something then unfocused it.